services/m3dbnode/config/cache.go

```diff
@@ -40,5 +40,6 @@ func (c CacheConfigurations) SeriesConfiguration() SeriesCacheConfiguration {
 
 // SeriesCacheConfiguration is the series cache configuration.
 type SeriesCacheConfiguration struct {
-	Policy series.CachePolicy `yaml:"policy"`
+	Policy       series.CachePolicy `yaml:"policy"`
+	MaxLRUBlocks uint               `yaml:"max_lru_blocks"`
```

Comment:
To make this a little more consistent for if we add future options for each individual policy we can make it it a nullable composite struct for each policy?

i.e.

Reply:
yeah good idea